docs/src/api/class-browsertype.md

```diff
@@ -62,6 +62,11 @@ This methods attaches Playwright to an existing browser instance.
   - `logger` <[Logger]> Logger sink for Playwright logging. Optional.
   - `timeout` <[float]> Maximum time in milliseconds to wait for the connection to be established. Defaults to
     `30000` (30 seconds). Pass `0` to disable timeout.
+  - `protocol` <"playwright"|"cdp"> To use a Chrome DevTools Protocol endpoint, pass "cdp". Defaults to "playwright".
```

Comment:
I'd call this `cdp: boolean | undefined`, so that you only use it if you know what you are doing. Otherwise you have some other legit option in there.

Reply:
I am worried that we will want to put some other protocols in here in the future. WebDriver. Firefox cdp. WebInspector protocol for iPhones.

Reply:
What do you all think about ChromiumBrowser.connectOVerCDP(wsEndpoint, options)? Are we moving away from ChromiumBrowser it is ok?

Reply:
We have ChromiumBrowserContext but not ChromiumBrowser. I'd rather have a small option than a big method.